bin/rucio

```diff
@@ -2480,6 +2564,21 @@ can be found in ' + Color.BOLD + 'http://rucio.cern.ch/documentation/rse_express
     touch_parser.add_argument(dest='dids', nargs='+', action='store', help='List of space separated data identifiers.')
     touch_parser.add_argument('--rse', dest='rse', action='store', help="The RSE of the DIDs that are touched.").completer = rse_completer
 
+    # The add-lifetime-exception command
+    add_lifetime_exception_parser = subparsers.add_parser('add-lifetime-exception', help='Add an exception to the lifetime model.',
+                                                          formatter_class=argparse.RawDescriptionHelpFormatter, epilog='''Usage example
+"""""""""""""
+::
+
+    $ rucio add-lifetime-exception --inputfile myfile.txt --reason "Needed for my analysis" --expiration 2015-10-30
```

Comment:
I’m afraid that ‘needed for my analysis’ will be used literally. :smile:
I think we need a better example in the documentation which encourages the user to better express the need for the exception.

Reply:
:point_up_2: This is the only point that *should* be addressed.